sdk-base/src/main/java/com/mapbox/maps/MapboxLogger.kt

```diff
@@ -0,0 +1,27 @@
+@file:JvmName("MapboxLogger")
+
+package com.mapbox.maps
+
+import com.mapbox.common.Logger
```

Comment:
isn't this deprecated in common?

Reply:
it is but we should still use it, no replacement yet 😉